utils/create_inheritance.sh

```diff
@@ -4,36 +4,78 @@
 
-main() {
-  echo -e "downloading ..."
-  wget --quiet --continue https://download.molgeniscloud.org/downloads/vip/images/utils/vcf-inheritance-3.0.1.sif
-  wget --quiet --continue https://download.molgeniscloud.org/downloads/vip/resources/utils/incomplete_penetrantie_genes_entrez_20210125.tsv
-  wget --quiet --continue http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa
-  wget --quiet --continue https://research.nhgri.nih.gov/CGD/download/txt/CGD.txt.gz
-  echo -e "downloading done"
+usage() {
+  echo -e "usage: ${SCRIPT_NAME} [-i <arg>]
+  -i, --input <arg>  path to omim genemap2 file
+
+  If no parameters are provided the inheritance file is created without OMIM."
+}
 
-  # create dummy genemap2.txt
-  local geneMapFilePath="genemap2.txt"
+createDummyGenemap() {
+  echo "create dummy ${geneMapFilePath}"
   cat >"${geneMapFilePath}" <<EOT
 # header line 1
 # header line 2
 # header line 3
 # Chromosome	Genomic	Position	Start	Genomic	Position	End	Cyto	Location	Computed	Cyto	Location	MIM Number	Gene Symbols	Gene Name	Approved Symbol	Entrez Gene ID	Ensembl Gene ID	Comments	Phenotypes	Mouse Gene Symbol/ID
 EOT
+  echo "create dummy ${geneMapFilePath} done"
+}
+
+main() {
+  local args=$(getopt -a -n pipeline -o ih --long input_omim:,help -- "$@")
+
+  local geneMapFilePath
+  eval set -- "${args}"
+  while :; do
+    case "$1" in
+    -h | --help)
+      usage
+      exit 0
+      ;;
+    -i | --input_omim)
+      geneMapFilePath="${2}"
+      shift 2
+      ;;
+    --)
+      shift
+      break
+      ;;
+    *)
+      usage
+      exit 2
+      ;;
+    esac
+  done
+
+  echo -e "downloading ..."
+  wget --quiet --continue https://download.molgeniscloud.org/downloads/vip/images/utils/vcf-inheritance-3.1.0.sif
+  wget --quiet --continue https://download.molgeniscloud.org/downloads/vip/resources/utils/incomplete_penetrantie_genes_entrez_20210125.tsv
+  wget --quiet --continue http://purl.obolibrary.org/obo/hp/hpoa/phenotype.hpoa
+  wget --quiet --continue https://research.nhgri.nih.gov/CGD/download/txt/CGD.txt.gz
+  echo -e "downloading done"
+
+  # create dummy genemap2.txt if not provided
+  if [ -z "${geneMapFilePath}" ]; then
+    geneMapFilePath="genemap2.txt"
+    createDummyGenemap "${geneMapFilePath}"
+  else
+    echo "genemap2 file provided, skipping dummy file creation."
+  fi
 
   local outputPath="inheritance_$(date '+%Y%m%d').tsv"
 
   local args=()
   args+=("-jar" "/opt/vcf-inheritance/lib/genemap-mapper.jar")
-  args+=("-i" "genemap2.txt")
+  args+=("-i" "${geneMapFilePath}")
   args+=("-h" "phenotype.hpoa")
   args+=("--incomplete_penetrance" "incomplete_penetrantie_genes_entrez_20210125.tsv")
   args+=("-c" "CGD.txt.gz")
   args+=("-o" "${outputPath}")
   args+=("-f")
 
   echo -e "creating ${outputPath} ..."
-  APPTAINER_BIND="${SCRIPT_DIR}" apptainer exec vcf-inheritance-3.0.1.sif java "${args[@]}"
+  APPTAINER_BIND="${SCRIPT_DIR},${geneMapFilePath}" apptainer exec vcf-inheritance-3.1.0.sif java "${args[@]}"
```

Comment:
`WARNING: While bind mounting '<cut>': destination is already in the mount point list` if `geneMapFilePath` is located in `SCRIPT_DIR`. How about removing the `APPTAINER_BIND` from the script altogether and leave it up to the callee to ensure that the binds are properly configured?